packages/rocket-uploads3-store-event-aws-infrastructure/src/lambdas/index.ts

```diff
@@ -0,0 +1,44 @@
+import { AWSError, DynamoDB } from 'aws-sdk'
+import { PutItemOutput } from 'aws-sdk/clients/dynamodb'
+
+export const handler = async (event: any): Promise<any> => {
+  const { v4: uuidv4 } = require('uuid')
+  const ddb = new DynamoDB.DocumentClient()
+
+  const {
+    bucket: { name },
+    object: { key, size },
+  } = event.Records[0].s3
+  const fileSize = size
+  const s3uri = `s3://${name}/${key}`
+
+  const boosterEvent = {
+    s3uri: s3uri,
```

Comment:
Could you please change this to `fileURI`? The idea is not to expose provider-aware names.

Also, Could you please create an exported type (typescript interface) for this so that people can use it in the definitions of their events?